### ISSUE TITLE
No vis when music runs on remote maschine

### ISSUE THREAD

**Issue by DenisMtfl** [dvx/lofi#129]
There is no vis when the music runs on a remote maschine so i think its hard to vis the sound because there is no sound on the local maschine. There should be a warning "no vis available because you are running the music on another maschine"

**Comment by stamoun** [comment]
Good point, there should be some feedback to the user when the sound cannot be read.

**Comment by stamoun** [comment]
Somewhat related to #135

**Comment by stamoun** [comment]
Could be fix by using spotify's track analysis instead of relying on the machine's output volume... This would make things a lot nicer, no more dependencies on node-gyp :)

https://developer.spotify.com/console/get-audio-analysis-track/
